### ISSUE TITLE
Create tests for core Slate JS files

### ISSUE THREAD

**Issue by t-kelly** [Shopify/slate#180]
### Problem

Slate includes a number of JS files that contain helpful functions that help developers with specific constraints in the Shopify theme development environment, i.e. `a11y.js`, `currency.js`, `images.js`, etc. These files should have tests to avoid any unexpected behaviour from future refactoring.

### Scope

Tests are useful for any code that needs an extra level of stability, such as code that will be used and depended on by other code. The Slate JS files are a great example of this.
